jupyter/d2l-java/chapter_convolutional-neural-networks/lenet.ipynb

```diff
@@ -402,68 +389,50 @@
     "    Loss loss = Loss.softmaxCrossEntropyLoss();\n",
     "\n",
     "    LearningRateTracker lrt = LearningRateTracker.fixedLearningRate(lr);\n",
-    "    Optimizer sgd = Optimizer.sgd().setLearningRateTracker(lrt).setRescaleGrad(1.0f/batchSize).build();\n",
```

Comment:
Just to clarify: You might see some of these rescaleGrad by 1.0/batchSize in the book. You don't have to worry about it as much in DJL because we take the mean in the loss instead of the sum. Either method is to ensure your loss is constant with respect to the batch size. We just divide within the loss instead of the optimizer

Reply:
Yes, I have fixed those :)